e2e/stack.go

```diff
@@ -243,8 +244,11 @@ func (s *Stack) runMonomer(ctx context.Context, env *environment.Env, genesisTim
 	env.DeferErr("close tx db", txdb.Close)
 	mempooldb := dbm.NewMemDB()
 	env.DeferErr("close mempool db", mempooldb.Close)
-	ethstatedb := rawdb.NewMemoryDatabase()
-	env.DeferErr("close eth state db", ethstatedb.Close)
+	rawDB := rawdb.NewMemoryDatabase()
+	env.DeferErr("close raw db", rawDB.Close)
```

Comment:
nit: It may be worth renaming `rawdb` to something like `rawstatedb` for readability, lmk what you think. This change would also apply in `integrations.go`

Reply:
I think the raw db is more of a generic db backend, not specific to state. Probably better to keep it as `rawdb`.

Reply:
Yeah, I think `rawdb` is fine too. But should it be in camelCase, `rawDB`, or `rawdb`?

Reply:
I don't have a strong opinion either way but we should use the same naming convention for all of the databases. It looks like we already have `txdb` and `mempooldb` as lowercase but it also looks like `blockPebbleDB` was recently added. Up to you on whether to rename

Reply:
Yep, deleted my previous comment after noticing the convention. I'll rename it to `rawdb` and push the changes

Thanks @natebeauregard

Reply:
Unfortunately, this causes a lint error:

```bash
e2e/stack.go:247:2: importShadow: shadow of imported from 'github.com/ethereum/go-ethereum/core/rawdb' package 'rawdb' (gocritic)
        rawdb := rawdb.NewMemoryDatabase()
        ^
integrations/integrations.go:203:2: importShadow: shadow of imported from 'github.com/ethereum/go-ethereum/core/rawdb' package 'rawdb' (gocritic)
        rawdb, err := rawdb.NewPebbleDBDatabase(
        ^
make: *** [lint] Error 1
```

I have to rename it back to `rawDB`

Reply:
Sounds good. You could alternatively use a different package reference for `rawdb` if you're getting a shadowing error but it's not that big of a deal